packages/flutter/lib/src/widgets/title.dart

```diff
@@ -17,13 +17,13 @@ class Title extends StatelessWidget {
   /// [color] and [child] are required arguments.
   Title({
     super.key,
-    this.title = '',
+    this.title,
```

Comment:
I think I would leave the Title widget itself unchanged (it's kinda strange that the title widget has a mode where it doesn't set a title) and instead modify the implementation in WidgetsApp to exclude the Title widget if WidgetsApp.title is null.

Reply:
Makes sense. Done!